autoload/sy/util.vim

```diff
@@ -242,11 +242,11 @@ else
 
     let signlist = execute('sign place buffer='. buf)
     for signline in split(signlist, '\n')[2:]
-      let tokens = matchlist(signline, '\v^\s+\S+\=(\d+)\s+\S+\=(\d+)\s+\S+\=(.*)\s+\S+\=(\d+)$')
+      let tokens = matchlist(signline, '\v^\s+\S+\=(\d+)\s+\S+\=(\d+)\s+\S+\=(.*)%(\s+\S+\=(\d+))\=$')
```

Comment:
Two modifications:

- regex should end with `=` instead of `\=`, because of `\v`
- `(.*)` should be changed to `(.{-})` to make match non-greedy